tests/test_helpers.py

```diff
@@ -860,6 +833,26 @@
 #      "List of all tags in citations collection:\n['nomonth', 'pdf']\nbuilding lists for all tags in the citation collection\nnomonth has been added/updated in reading_lists\npdf has been added/updated in reading_lists\n"),
 #     ]
 
+helper_map_bad = [
+ (["helper", "u_milestone", "--milestone_uuid", "sb_fir", "--projectum_id", "sb_firstprojectum",
+   "--due-date", "2023-01-01", "--notes", "do this", "do that", "--type", "mergedpr"],
+  "A uuid fragment and projectum id have been entered.\n"
```

Comment:
To help the user,
`Detected both a uuid fragment and a projectum id.`

Then below `You may enter either a milestone uuid....`

Reply:
👍